unittests/svnn_ibc_tests.cpp

```diff
@@ -575,4 +607,57 @@ BOOST_AUTO_TEST_SUITE(svnn_ibc)
 
    } FC_LOG_AND_RETHROW() }
 
+   BOOST_AUTO_TEST_CASE(bitset_tests) { try {
+
+      savanna_tester chain;
+
+      chain.create_account( "ibc"_n );
+      chain.set_code( "ibc"_n, eosio::testing::test_contracts::ibc_wasm());
+      chain.set_abi( "ibc"_n, eosio::testing::test_contracts::ibc_abi());
+
+      std::string bitset_1 = binary_to_hex("0");
+      std::string bitset_2 = binary_to_hex("011");
+      std::string bitset_3 = binary_to_hex("00011101010");
+      std::string bitset_4 = binary_to_hex("11011000100001");
+      std::string bitset_5 = binary_to_hex("111111111111111111111");
+      std::string bitset_6 = binary_to_hex("000000111111111111111");
+
+      chain.push_action("ibc"_n, "testbitset"_n, "ibc"_n, mvo()
+         ("bitset_string", "00")
+         ("bitset_vector", bitset_1)
+         ("finalizers_count", 1)
+      );
+
+      chain.push_action("ibc"_n, "testbitset"_n, "ibc"_n, mvo()
+         ("bitset_string", "30")

```

Comment:
Maybe you can add a comment what those parameters are testing, like "30" is the reverse of bitset_2 or something.